tests/testthat/test-cpp-distributions.R

```diff
@@ -51,19 +51,21 @@ test_that("pwexp", {
   pwexp <- new(PwExp, rate = rate, time = time)
   
   # pdf
-  expect_error(pwexp$pdf(3))
+  expect_equal(pwexp$pdf(5.5), 1.2*exp(-(5.0*0.8 + 0.5*1.2)))
```

Comment:
We should test the edge case and confirm that we get the same answer as with `msm`:
```suggestion
  expect_equal(pwexp$pdf(5.5), 1.2*exp(-(5.0*0.8 + 0.5*1.2)))
  
  expect_equal(pwexp$pdf(5), 1.2*exp(-(5.0*0.8)))
  expect_equal(pwexp$pdf(5), 1.2*exp(-(5.0*0.8)))
```